plugins/macro.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: macro
+spec:
+  homepage: https://github.com/morningspace/kubemacro
+  shortDescription: "Wrap a set of kubectl calls into one command and run as a macro."
```

Comment:
I think you need to make this shorter, it probably doesn't fit as it is now. Please eliminate redundant words.

Reply:
Sure, it is revised now, would it look better?